imageio/plugins/ffmpeg.py

```diff
@@ -91,33 +115,39 @@ def get_exe():
         try:
             with open(os.devnull, "w") as null:
                 sp.check_call([exe, "-version"], stdout=null, stderr=sp.STDOUT)
-                return exe
+                exes.append(exe)
```

Comment:
I guess this should be `insert`?

Maybe its easiest to just append to `exes`, in the order of priority, and add `'ffmpeg'` last. 

I think it would be ok to give highest priority to the one provided by imageio. But it would be good to have a way to remove the binary to have more control, e.g. in case its corrupt. Can you add that to the CLI?